.github/workflows/pull-request-checks.yml

```diff
@@ -101,11 +102,6 @@ jobs:
             - name: Install dependencies
               if: steps.cache.outputs.cache-hit != 'true'
               run: yarn install --frozen-lockfile
-            - name: Run Jest tests with coverage report
-              with:
-                  skip-step: install
-              uses: ArtiomTr/jest-coverage-report-action@v2
-              if: github.event_name == 'pull_request'
```

Comment:
We will be migrating these tests to Drone and losing this feature anyway. It was failing for some reason so I just took it out.